indico/modules/rb_new/client/js/common/bookings/BookingDetailsModal.jsx

```diff
@@ -0,0 +1,263 @@
+            const details = (info[1] ? info[1] : null);
+            const logDate = serializeDate(toMoment(timestamp), 'L');
+            if (details) {
+                const popupContent = <span styleName="popup-center"> {details}</span>;
+                const wrapper = <PopupParam content={popupContent} />;
+                return (
+                    <List.Item key={i}>
+                        <Translate>
+                            <Param name="date" value={logDate} wrapper={<span styleName="log-date" />} />
+                            {' - '}
+                            <Param name="info" wrapper={wrapper} value={basicInfo} /> by
+                            {' '}
+                            <Param name="user" value={userName} />
+                        </Translate>
+                    </List.Item>
+                );
+            }
+            return (
+                <List.Item key={i}>
+                    <Translate>
+                        <Param name="date" value={logDate} wrapper={<span styleName="log-date" />} />
+                        {' - '}
+                        <Param name="details" value={basicInfo} /> by
+                        {' '}
+                        <Param name="user" value={userName} />
+                    </Translate>
+                </List.Item>
+            );
+        }));
+        return (
+            <div styleName="booking-logs">
+                <Header><Translate>Booking history</Translate></Header>
+                <List divided styleName="log-list">
+                    {items}
+                    <List.Item>
+                        <Translate>
+                            <Param name="date" value={createdDate} wrapper={<span styleName="log-date" />} />
+                            {' - '}
+                            <Param name="details" wrapper={<strong />}>Booking created</Param> by
+                            {' '}
+                            <Param name="created-by" value={createdBy} />
+                        </Translate>
+                    </List.Item>
+                </List>
+            </div>
+        );
+    };
+
+    renderBookedFor = (bookedForUser) => {
+        if (bookedForUser) {
+            const {fullName: bookedForName, email: bookedForEmail, phone: bookedForPhone} = bookedForUser;
+            return (
+                <>
+                    <Header><Icon name="user" /><Translate>Booked for</Translate></Header>
+                    <div>{bookedForName}</div>
+                    {bookedForPhone && <div><Icon name="phone" />{bookedForPhone}</div>}
+                    {bookedForEmail && <div><Icon name="mail" />{bookedForEmail}</div>}
+                </ >
+            );
+        }
+    };
+
+    renderReason = (reason) => (
+        <Message info icon styleName="message-icon">
+            <Icon name="info" />
+            <Message.Content>
+                <Message.Header>
+                    <Translate>Booking reason: </Translate>
```

Comment:
do we really want the `:` there? also, let's not put trailing whitespace in translate - we strip it anyway.